### DISCUSSION TITLE
Image tag src with ip adress not render image

### DISCUSSION THREAD

**Discussion by smartdeveler** [vercel/next.js#70506] [App Router]
When i pass src (such as http://192.168.11.8/images/cam1.png) with ip adress image not render and behave such local src in Image tag.
I add remotePatterns in next config but still not render image.

**Comment by Tiimie1** [comment]
Check if the server  is allowing CORS from webapp origin. If the server hosting the images is not configured to allow requests from other origins, the image won't load.

**Comment by icyJoseph** [comment]
Are there any logs? in the browser network, you can see the failed image request, and it typically contains some information. 

Also that IP is a LAN IP right? Maybe you can't even reach the image to begin with?
